### PR TITLE
fix(website): react key error on internal pages of website

### PR DESCRIPTION
**Pull request by y-hsgw**
## PR Checklist

- [x] Addresses an existing open issue: fixes https://github.com/typescript-eslint/typescript-eslint/issues/9333
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

It appears that the error occurred due to the presence of a key in the previous version of `LineInputProps`. This issue has been resolved by updating the prism-react-renderer version.
※I determined that the impact area is limited, as prism-react-renderer is used only in `packages/website/src/theme/CodeBlock/Content/String.tsx`. Therefore, I proceeded with a major update. Please advise on the appropriate flow for updating package versions if there is a recommended procedure.

### CODE REVIEW

**Review by Josh-Cena — APPROVED** [review]
Effect LGTM

File: yarn.lock
Comment (JoshuaKGoldberg) [inline yarn.lock:1]: [Praise] Always great to see a deduplication 😄 nice!

**Review by JoshuaKGoldberg — APPROVED** [review]
Confirmed this fixes it for me, and both the existing syntax highlighting and playground still work. Thanks for the cleanup! 🧹

**Comment by JoshuaKGoldberg** [comment]
Oh, dangit, I should have renamed this to `chore: ` or `chore(website): `. #5218

**Pull request merged**